pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -195,7 +195,7 @@ def __init__(
         monitor: Optional[str] = None,
         verbose: bool = False,
         save_last: Optional[bool] = None,
-        save_top_k: Optional[int] = None,
+        save_top_k: int = 1,
```

Comment:
a mock call in the wandb test needs to update too :)

Reply:
Should it be 0 by default to match previous behaviour ?